search-sentiments-map/public/scripts/trends-script.js

```diff
@@ -75,3 +76,10 @@ function showMoreOrLess() {
     showMoreOrLessToggleItem.innerText = TOGGLE_SHOW_MORE;
   }
 }
+
+/** */
```

Comment:
Need JSDoc for this.

Reply:
This function is not yet complete but yeah I can put a TODO